### ISSUE TITLE
[Bug] Wo ist jetzt die Leistungseinstellung ab 0.8.36

### ISSUE THREAD

**Issue by kiu77** [lumapu/ahoy#1464]
Bei den neueren Versionen finde ich peiplicherweise die Einstellung der Leistung nicht mehr.
Ich stehe wahrscheinlich mit beiden Beinen auf der Leitung.
Schubst mich jemand runter?

**Comment by knickohr** [comment]
Guckst Du da 😉

![IMG_1976](https://github.com/lumapu/ahoy/assets/63048210/5b0b9c6d-33b3-4b92-b91c-7414cb810354)

**Comment by kiu77** [comment]
Vielen herzlichen Dank!
Ich bin einfach nicht drauf gekommen, dass sich an der Stelle eine Schaltfläche verbirgt.
Das sollte man vielleicht farblich akzentuieren, damit es auffällt.

**Comment by mroenne2022** [comment]
@kiu77 
Das sind tatsächlich 3 Schaltflächen von links , ausser Temp. nicht. Das war mir auch noch nicht aufgefallen , von daher hat dein Issue mir auch geholfen :-) Vielen Dank für die Info @knickohr ;-)

**Comment by kiu77** [comment]
Vorschlag, wie es aussehen könnte:

![Bildschirmfoto 2024-03-01 um 14 24 26](https://github.com/lumapu/ahoy/assets/43971413/b31bd494-7abd-4b75-ba0d-0f3907093213)

**Comment by kiu77** [comment]
Oder so:

![Bildschirmfoto 2024-03-01 um 14 24 26](https://github.com/lumapu/ahoy/assets/43971413/b80f9632-145d-4b66-825c-7767085370c1)

**Comment by MetaChuh** [comment]
@kiu77 

> Das sollte man vielleicht farblich akzentuieren, damit es auffällt.

idealerweise neon pink, und je mehr % eingestellt sind, desto schneller stroboskop-blinkend ... disco disco 😃

**Comment by kiu77** [comment]
;-)
Jaja...
Aber zugeben würdest Du, dass das in der aktuellen Form etwas subliminal für die Wahrnehmung ist.
Bei den Einstellungsseiten ist ja auch klar, wo ne Schaltfläche ist.
Ist fast wie hier: [DER UNSICHTBARE Trailer German Deutsch (2020)](https://www.youtube.com/watch?v=6DZ4x75MPNA)

**Comment by knickohr** [comment]
Das ist die Dau-Sicherung 🤪🫣

Aua, nicht hauen 🙉🙈🙊

**Comment by kiu77** [comment]
Der war gut ☺
Aber diesen DAU hier hat es ja nicht wirklich abschrecken können...

**Comment by lumapu** [comment]
Ich finde die Frage nicht peinlich sondern berechtigt, hier müssen wir uns was schönes einfallen lassen.

**Comment by firedp** [comment]
Ich hatte vor Wochen ja die gleiche Frage 
😂

**Comment by MetaChuh** [comment]
beispiel mit regler schaltfläche (mock-up):
etwa so ?

<img width="871" alt="Bildschirmfoto 2024-03-02 um 08 11 17" src="https://github.com/lumapu/ahoy/assets/45773305/03dc8921-aeb2-41ec-bca7-6d99f2eda3b0">

-----

> @kiu77 : Ist fast wie hier: [https://www.youtube.com/watch?v=6DZ4x75MPNA](https://github.com/lumapu/ahoy/issues/url)

hahaha, so ein psychoterror ist die schaltfläche sicher nicht 🙈😂

**Comment by ralfberger** [comment]
würde mir bitte jemand sagen was ich unter 
Control : Restart  /  Turn off  /  Turn on
verstehen soll?
Bei Leistungsüberschreitung  Restart oder  Aus- bzw. Einschalten??

**Comment by kiu77** [comment]
@ralfberger 
???
Was, wer, wie, wo?

**Comment by knickohr** [comment]
Man kann den WR hier ein- und ausschalten, oder rebooten.

Und nein, nicht bei Überschreitung. Immer.

**Comment by ralfberger** [comment]
Sorry wenn ich mich unklar ausgedrückt habe.
Das war gemeint.

![Ahoi2](https://github.com/lumapu/ahoy/assets/63057484/b9a2a0a9-1fe7-4319-ba19-7c8ad9d2aa4a)

**Comment by ralfberger** [comment]
O.K. ich dachte es bezieht sich aufs Limit .

**Comment by DanielR92** [comment]
Kann man dieses Issue schließen?

**Comment by ralfberger** [comment]
ich denke schon!

**Comment by stefan123t** [comment]
@DanielR92 nein, denn die Buttons / Funktionen sind immer noch nicht als solche erkenntlich:

Das ist m.E. schlechtes UX Design und auf Arbeit bekäme ich beim Barrierefreiheits-Test Probleme wenn die Elemente nicht klar erkennbar und fokusierbar sind. Die anderen Buttons z.B. unter Settings sind ja auch alle Blau mit runden Ecken.

**Success Criterion 3.2.7: Visible Controls**
https://w3c.github.io/wcag/understanding/visible-controls.html

> Oder so:
> 
> ![Bildschirmfoto 2024-03-01 um 14 24 26](https://private-user-images.githubusercontent.com/43971413/309309161-b80f9632-145d-4b66-825c-7767085370c1.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzAyNDgxNjUsIm5iZiI6MTczMDI0Nzg2NSwicGF0aCI6Ii80Mzk3MTQxMy8zMDkzMDkxNjEtYjgwZjk2MzItMTQ1ZC00YjY2LTgyNWMtNzc2NzA4NTM3MGMxLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMzAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDMwVDAwMjQyNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTRhMTVmY2NkYmYzNmViYzA3NTQzMDk0NjNjYjU2ZjgwNmY3NmFlZmJlYTI0OTc2ZDg4YWFjYWNiNGE0N2ZjZTUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.m8p7R-4CV7RM71zxDj1N2V_rDDt93LruIDMqXHbOhxI)
